ffmpeg/lpms_ffmpeg.c

```diff
@@ -649,7 +650,8 @@ int lpms_transcode(char *inp, output_params *params, int nb_outputs)
     if (params[i].fps.den) octx->fps = params[i].fps;
     if (ictx.vc) {
       char filter_str[256];
-      snprintf(filter_str, sizeof filter_str, "fps=fps=%d/%d,scale=%dx%d:force_original_aspect_ratio=decrease", octx->fps.num, octx->fps.den, octx->width, octx->height);
+      // preserve aspect ratio along the larger dimension when rescaling
+      snprintf(filter_str, sizeof filter_str, "fps=fps=%d/%d,scale='w=if(gte(iw,ih),%d,-2):h=if(lte(iw, ih),%d,-2)'", octx->fps.num, octx->fps.den, octx->width, octx->height);
```

Comment:
this will not preserve ration in case of square input. do you think it is ok or better to fix it?